content/developers/api-reference/events-api/index.md

```diff
@@ -391,6 +391,90 @@ You should see the response:
 }
 ```
 
+### Event Record Retrieval
+
+Event records in DataTrails are tokenized at creation time and referred to in all future API calls by a permanent unique identity of the form:
+
+```bash
+assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
```

Comment:
it's great we're obfuscating the asset_id's we use to create the docs. Can we create environment variables so users can copy/paste the commands? It's a bit confusing to know which guid's should be replaced, and when.